main.js

```diff
@@ -100,50 +27,112 @@ function renderItem(item) {
   $itemDescription.classList.add('card-text')
   $learnMore.classList.add('card-text')
   $button.classList.add('btn', 'btn-primary')
-  $image.setAttribute('src', item.image)
+  $button.setAttribute('href', item.addToCart)
```

Comment:
clicking on the `Add To Cart` button will take me to Google's homepage, which is probably an unintended feature. I would suggest to leave this blank, and them implement this in a future feature.